src/ContactHub.js

```diff
@@ -87,6 +87,36 @@ const cleanCustomer = (data: APICustomer): Customer => {
   return customer;
 };
 
+/**
+ * Helper for adding 'page' to args.params object
+ */
+
+const buildArguments = (args: Object, page: number): Object => ({
+  ...args,
+  params: {
+    ...args.params,
+    page: (args.params.page || 0) + page
+  }
+});
+
+/**
+ * Helper to create a recursive structure for paginated resources
+ */
+
+const buildPaginatedResource = (promise: Function, args: Object): Promise<Paginated<any>> => {
+  return promise(args).then(({ page: { number, totalPages }, elements }) => {
+    return {
+      page: {
+        current: number,
+        prev: () => number > 0 && buildPaginatedResource(promise, buildArguments(args, -1)),
+        next: () => number < totalPages && buildPaginatedResource(promise, buildArguments(args, 1)),
+        total: totalPages
+      },
+      data: elements
```

Comment:
I suggested calling this `data` in the issue, but thinking again maybe it makes sense to use `elements` to be consistent with the API terminology. What do you think?